source/renderer/app/components/wallet/transactions/Transaction.js

```diff
@@ -379,7 +379,7 @@ export default class Transaction extends Component<Props, State> {
               <div className={styles.details}>
                 <div className={styles.type}>
                   {intl.formatMessage(messages.type, { currency })},{' '}
-                  {moment(data.date).format(currentTimeFormat)}
+                  <em>{moment(data.date).format(currentTimeFormat)}</em>
```

Comment:
@daniloprates maybe we shouldn't change code that we want to test just to fit tests. Is this some requirement not related to tests?
If you need that for test and if is impossible to get that in scenario rather use React.Fragment than <em> which will transform text!

Reply:
✅